ncj/3dsmax/scripts/3dsmax.ps1

```diff
@@ -70,10 +70,33 @@ function SetupDistributedRendering
     $vrayLogFile = "$env:AZ_BATCH_TASK_WORKING_DIR\VRayLog.log" -replace "\\", "\\"
     $script:pre_render_script_content += "-- VRay DR setup`r`n"
     $script:pre_render_script_content += "rendererName = r as string`r`n"
-    $script:pre_render_script_content += "index = findString rendererName ""V_Ray_""`r`n"
+
     $script:pre_render_script_content += "if index != 1 then (print ""VRay renderer not used, please save the scene with a VRay renderer selected."")`r`n"
-    $script:pre_render_script_content += "index = findString rendererName ""V_Ray_RT_""`r`n"
-    $script:pre_render_script_content += "if index == 1 then (r.distributed_rendering = true) else (r.system_distributedRender = true;r.system_vrayLog_level = 4; r.system_vrayLog_file = ""$vrayLogFile"")`r`n"
+
+    If($maxVersion -eq "2018")
+    {
+        IF($renderer -eq "VRayRT")
+        {
+            echo "RT 2018"
+            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_RT_""`r`n"
+            $script:pre_render_script_content += "if index == 1 then (r.distributed_rendering = true)`r`n"
+
+        }
+        ElseIf($renderer -eq "VRayAdv")
+        {
+            echo "Adv 2018"
+            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_Adv""`r`n"
+            $script:pre_render_script_content += "if index == 1 then (r.system_distributedRender = true;r.system_vrayLog_level = 4; r.system_vrayLog_file = ""$vrayLogFile"")`r`n"
+
+        }
+    }
+    ElseIf($maxVersion -eq "2019"){
+        IF($renderer -eq "VRayRT"){
```

Comment:
Don't we need to support VRay Adv for 2019?

With 2019 do we support VRay 3.6 too?  Either way we'll need to support Adv.